atlasdb-dbkvs-tests/build.gradle

```diff
@@ -51,6 +51,7 @@ task postgresTest(type: Test) {
 task oracleTest(type: Test) {
     maxHeapSize = "1024m"
     include '**/DbKvsOracleTestSuite.class'
+    include '**/OracleNamespaceDeleterTests.class'
```

Comment:
I chose not to add it to the test suite, since that spins up and blocks on oracle containers spinning up. The unit tests explicitly and intentionally do not use oracle containers (instead, a sqlite adapter that I wrote to test oracle sql commands on SQLite for speed).